translations/ui-quick-marc/en.json

```diff
@@ -5,7 +5,7 @@
   "permission.quick-marc-editor.duplicate": "quickMARC: Derive new MARC bibliographic record",
 
   "record.edit.title": "Edit MARC record - {title}",
-  "record.duplicate.title": "Create a new MARC bib record",
+  "record.duplicate.title": " Derive a new MARC bib record",
```

Comment:
Do we need an empty space at the beginning?

Reply:
No, thanks for finding this